Source/WebCore/css/parser/CSSPropertyParserCustom.h

```diff
@@ -272,19 +272,6 @@ inline bool PropertyParserCustom::consumeStandardSpaceSeparatedShorthand(CSSPars
     return true;
 }
 
-inline bool PropertyParserCustom::consumeSingleShorthand(CSSParserTokenRange& range, PropertyParserState& state, const StylePropertyShorthand& shorthand, PropertyParserResult& result)
```

Comment:
You can remove this as well: https://searchfox.org/wubkat/rev/2326b7031d277e0dc380c41edb926888df928161/Source/WebCore/css/parser/CSSPropertyParserCustom.h#126